frontend/components/bc/table/BcTablePopoutEdit.vue

```diff
@@ -4,12 +4,13 @@ import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
 
 const emit = defineEmits<{ (e: 'onEdit'): void }>()
 
-interface Props {
+defineProps<{
+  disabled?: boolean,
```

Comment:
our naming convention is `boolean props` are starting with `is-` or `has-`
Understandably the question arises what is with `DOM attributes`: 
- they could be applied via `fallthrough`
- so if we specify a `public api via our own props` we should stick to our `conventions`
```suggestion
  isDisabled?: boolean,
```

Reply:
Changed to `isDisabled`